tests/common/helpers/platform_api/psu.py

```diff
@@ -78,6 +78,9 @@ def get_current(conn, psu_id):
 def get_power(conn, psu_id):
     return psu_api(conn, psu_id, 'get_power')
 
+def get_maximum_supplied_power(conn, psu_id):
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be two blank lines between top-level function definitions.